internal/runbits/runtime/progress/dotprogress.go

```diff
@@ -27,7 +27,7 @@ func NewDotProgressIndicator(out output.Outputer) *DotProgressDigester {
 
 func (d *DotProgressDigester) Handle(event events.Eventer) error {
 	switch event.(type) {
-	case events.Start:
+	case events.Start, events.SolveStart:
```

Comment:
```suggestion
	case events.SolveStart:
```

Having it trigger on both is going to trigger the logic twice. Looking at the interactive version of this code the spinner is starting on SolveStart, not Start. So just listening to SolveStart seems appropriate.